features/old/liquid/active_docs.feature

```diff
@@ -53,7 +53,25 @@ Feature: ActiveDocs
 
     <h3>ActiveDocs version 2.2.10</h3>
 
-    {% include 'shared/swagger_ui' %}
+    <div id="swagger-ui-container" class="swagger-ui-wrap"></div>
+
+    <script type="text/javascript">
+      (function () {
+        var url = "{{provider.api_specs.first.url}}";
+        window.swaggerUi = new SwaggerUi({
+          url: url,
+          dom_id: 'swagger-ui-container',
+          supportedSubmitMethods: ['get', 'post', 'put', 'delete', 'patch'],
+          apisSorter: 'alpha', // can also be a function
+          operationsSorter: 'method', // can also be 'alpha' or a function
+          onComplete: function(swaggerApi, swaggerUi) {
+            $(document.getElementById(swaggerUi.dom_id)).find('pre code').each(function(i, e) {hljs.highlightBlock(e)})
+          },
+          docExpansion: 'list',
+          transport: function (httpClient, obj) { ApiDocsProxy.execute(httpClient, obj) }
```

Comment:
This one is really equal to the `shared/swagger_ui` https://github.com/3scale/porta/blob/a903259ffabbf928e7694c94f9463c3737e813f2/lib/developer_portal/app/views/developer_portal/shared/_swagger_ui.html.liquid

Reply:
`swagger_ui` was modified at the beginning but I reverted it in the last commit, this is wrong and I'm changing it, thanks.